src/modules/app/containers/__tests__/Switcher.spec.js

```diff
@@ -284,6 +307,14 @@ describe('Switcher', () => {
 
       expect(switcher).toMatchSnapshot()
     })
+
+    it('is the pois route', () => {
```

Comment:
should be the pois route

Reply:
Nope, this is good, the `describe()` says `should get the right page if data has been fetched and`.